src/repository/opamRepository.mli

```diff
@@ -58,7 +58,8 @@ val pull_file:
   unit download OpamProcess.job
 
 (** Same as [pull_file], but without a destination file: just ensures the file
-    is present in the cache. *)
+    is present in the cache and that every listed checksums are correctly linked
+    to the archive. *)
```

Comment:
```suggestion
    is present in the cache and that every listed checksums are correctly linked
    to the archive, otherwise add missing links. *)
```

Reply:
done